src/features/validation/validate.ts

```diff
@@ -168,6 +171,24 @@ export class ApplyMarkersCommand extends FeedbackCommand {
     }
 }
 
+function addCSSClassToIssueParent(modelElement: SParentElement, issueMarker: SIssueMarker) {
```

Comment:
You could reuse the utility mehtods for adding/removing cssClasses for this: https://github.com/eclipse-glsp/glsp-client/blob/db93ed9d9e6fa8926d662c45081edcc233d4de98/src/utils/smodel-util.ts#L61

Reply:
Thanks, addressed in a follow-up commit